schema.json

```diff
@@ -0,0 +1,514 @@
+{
+  "$schema": "http://json-schema.org/LivingDocumentation",
+  "title": "Intermediary JSON for LivingDocumetation",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "FullName": {
+        "type": "string"
+      },
+      "Type": {
+        "type": "number"
```

Comment:
`"number"` seems a bit broad here - it includes negative numbers, floating-point numbers, and so on. Do we want to be more specific? What does the C# implementation do when it's fed something unexpected? Do we want to pin it all the way down to `"enum": [0, 1, 2, 3]` or maybe `"type": "integer", "minimum": 0"`?

Same for `Modifiers`, below and elsewhere.